GPU/GLES/GLES_GPU.cpp

```diff
@@ -731,6 +736,10 @@ void GLES_GPU::BeginFrameInternal() {
 		dumpThisFrame_ = false;
 	}
 
+	// Save the cache from time to time. TODO: How often?
```

Comment:
Probably won't be bad to do it this often, although I guess on a desktop if the disk goes to sleep it could cause a hitch.  If we care about that, maybe we do the actual IO on a lightweight thread `new std::thread`?

-[Unknown]